File: python-sdk/src/planetscaleapi/models/operations/get_a_database.py
import dataclasses
from typing import Any,Optional


@dataclasses.dataclass
class GetADatabasePathParams:
    name: str = dataclasses.field(metadata={'path_param': { 'field_name': 'name', 'style': 'simple', 'explode': False }})
    organization: str = dataclasses.field(metadata={'path_param': { 'field_name': 'organization', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class GetADatabaseRequest:
    path_params: GetADatabasePathParams = dataclasses.field()
    

@dataclasses.dataclass
class GetADatabaseResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    get_a_database_200_application_json_object: Optional[dict[str, dict[str, Any]]] = dataclasses.field(default=None)
    

File: go-sdk/pkg/models/operations/getadatabase.go
package operations

type GetADatabasePathParams struct {
	Name         string `pathParam:"style=simple,explode=false,name=name"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type GetADatabaseRequest struct {
	PathParams GetADatabasePathParams
}

type GetADatabaseResponse struct {
	ContentType                          string
	StatusCode                           int
	GetADatabase200ApplicationJSONObject map[string]map[string]interface{}
}
